nfafrontend/src/app/current-project/project-edit/project-edit.component.html

```diff
@@ -47,9 +47,15 @@
             <label for="branch"><small>{{'nfa.new-project.branch.label' | translate }}</small></label>
             <select  id="branch" formControlName="branch" class="form-control">
               <option value="" disabled>{{'nfa.new-project.branch.choose' | translate }}</option>
-              <option value="Public Sector">{{'nfa.new-project.branch.public-sector' | translate }}</option>
-              <option value="Automotive">{{'nfa.new-project.branch.automotive' | translate }}</option>
-              <option value="Banking">{{'nfa.new-project.branch.banking' | translate }}</option>
+              <option value="Automotive">{{'nfa.new-project.branch.automotive'}}</option>
+              <option value="Banking">{{'nfa.new-project.branch.banking'}}</option>
+              <option value="Food">{{'nfa.new-project.branch.food'}}</option>
+              <option value="Insurance">{{'nfa.new-project.branch.insurance'}}</option>
+              <option value="Public Sector">{{'nfa.new-project.branch.public-sector'}}</option>
+              <option value="Life Science & Healthcare">{{'nfa.new-project.branch.life-science-healthcare'}}</option>
+              <option value="Telecommunications">{{'nfa.new-project.branch.telecommunications'}}</option>
+              <option value="Travel & Logistics">{{'nfa.new-project.branch.travel-logistics'}}</option>
+              <option value="Utilities">{{'nfa.new-project.branch.utilities'}}</option>
             </select>
```

Comment:
![image](https://user-images.githubusercontent.com/38284358/40946141-502e292c-685c-11e8-9727-d47c4e3d376d.png)

I think this problem occurs when we donot add "translate" in code. Could you please correct.